content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,21 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/depends-on
+
+Type: Annotation
+
+Example: `config.kubernetes.io/depends-on: "OBJECT_REFERENCE"` 
```

Comment:
This doesn't look right - is that a valid actual example?

Reply:
@sftim should I mention this in the example value?
```
annotations:
  config.kubernetes.io/depends-on: |
    /namespaces/default/Pod/pod-a,
    /namespaces/default/Pod/pod-c

```